appveyor.yml

```diff
@@ -3,8 +3,8 @@ clone_depth: 1
 clone_folder: c:\projects\symfony
 
 cache:
-    - c:\projects\symfony\composer.phar
-    - .phpunit -> phpunit
+    - composer.phar
+    - .phpunit -> vendor/symfony/phpunit-bridge/bin/simple-phpunit
```

Comment:
this is broken, because ``vendor/symfony/phpunit-bridge/bin/simple-phpunit`` is not committed.

You should just cache it unconditionally as simple-phpunit already takes care of reinstalling PHPUnit when updated